lib/hive/windevice.rb

```diff
@@ -0,0 +1,44 @@
+require 'hive/device'
+
+module Hive
+  # The generic device class
+  class Windevice < Device
```

Comment:
There is a problem with having a separate class for Windows devices. The Device class is the parent for the device classes in Hive Runner modules and so for transparency it should be possible to only inherit from Hive::Device and then leave it to the core Hive Runner to deal with the underlying operating system. With a separate Windevice class it is necessary for the module to duplicate its child classes and then, potentially, have logic in the controller child module accordingly.